package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "winners-c8-bn-be",
+  "version": "1.0.0",
+  "main": "test.js",
+  "scripts": {
```

Comment:
@pacomesimon 
Can you add the dev script  that use the nodemon

Reply:
I have added it (it uses `nodemon`, as you had requested before)